src/Resources/config/services/validator.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is this file included in the root `services.xml`?

Reply:
No it is not, but for some reasons it works fine.
I'm adding it